java/core/src/main/java/co/worklytics/psoxy/rules/asana/PrebuiltSanitizerRules.java

```diff
@@ -23,9 +23,12 @@ public class PrebuiltSanitizerRules {
                     .jsonPath("$.data[*].photo")
                     .build())
             .transform(Transform.Pseudonymize.builder()
-                    .jsonPath("$.data[*].gid")
                     .jsonPath("$.data[*].email")
                     .build())
+            .transform(Transform.Pseudonymize.builder()
+                .includeReversible(true)
```

Comment:
Is it reversible only for `user` endpoints, or does it needs to be reversible for any `gid` belong to the user? Because in that case, that needs to be included in other rules too

Reply:
no, just for user endpoints (where we might need it as arg to some follow-up call)